Resources/Views/index.leaf

```diff
@@ -262,9 +262,9 @@
 
         <p>You should be aware that using a custom <tt>dateFormat</tt> comes with a risk of falling into some fallacies.</p>
         <p>
-            Especially, you need to realize that the user can use different <tt>Locales</tt>,
-            and that date formats are different for different locales, regions and user settings.
-            For example one date formatted with a <tt>dateFormat</tt> that fits the US might become confusing anyone in for Europe.
+          Especially, you need to realize that the user can use different <tt>Locales</tt>,
+          and that date formats are different for different locales, regions and user settings.
+          For example one date formatted with a <tt>dateFormat</tt> that fits the US might become confusing anyone in for Europe.
```

Comment:
> become confusing anyone in for Europe
"become confusing for anyone in Europe" you mean?

Reply:
indeed.
```suggestion
          For example one date formatted with a <tt>dateFormat</tt> that fits the US might become confusing for anyone in Europe.
```